src/parser/codeParser.ts

```diff
@@ -5,11 +5,12 @@ const testTokens = ["suite", "describe", "context", "it", "specify", "test"];
 function codeParser(sourceCode) {
   const parserOptions: ParserOptions = {
     plugins: [
-      "jsx",
       "typescript",
-      "objectRestSpread",
-      "optionalChaining",
-      "nullishCoalescingOperator"
+      ["decorators", { decoratorsBeforeExport: false }],
```

Comment:
❗ The plugins that already exist should be maintained

Reply:
I removed any plugin that is enabled by default. I hope that is okay [See babel parser notes](https://babeljs.io/docs/en/babel-parser#latest-ecmascript-features)
As the plugin ages and the parser is updated, more language features are supported by default.